DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,21 @@
    "metadata": {},
```

Comment:
<p>Would suggest changing "is then used" to "could be used", since you don't actually do it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>